### PR TITLE
Cluster Identity Resource Implemented

### PR DESCRIPTION
**Pull request by Sakshi-dell**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Feature Pull Request

##### RESOURCE OR DATASOURCE NAME

Cluster identity resource

##### OUTPUT

image

##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Unit tests
- [x] Acceptance tests

### CODE REVIEW

**Comment by Sakshi-dell** [comment]
run e2e test TestAccClusterIdentity

**Comment by Sakshi-dell** [comment]
run e2e test ClusterIdentity

File: examples/resources/powerscale_cluster_identity/resource.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# `terraform destroy` will delete the resource from terraform state file rather than deleting identity Settings from PowerScale.
+# For more information, Please check the terraform state file.
+
+# PowerScale Cluster identity Settings allow you to change the settings of identity
+resource "powerscale_cluster_identity" "example_identity_settings" {
+  # Optional fields both for creating and updating
+  name        = "cluster1"
+  description = "cluster1 description"
+  logon = {
+    motd = "motd", motd_header = "motd header"
```
Comment (shenda1) [inline examples/resources/powerscale_cluster_identity/resource.tf:30]: Can you elaborate on these attributes?

**Review by shenda1 — APPROVED** [review]

**Review by rounak-adhikary — APPROVED** [review]

**Comment by Sakshi-dell** [comment]
run e2e test ClusterIdentity

**Comment by Sakshi-dell** [comment]
run e2e test ClusterIdentity

**Comment by Sakshi-dell** [comment]
run e2e test ClusterIdentity

**Pull request merged**